examples/speech_recognition/tasks/speech_recognition.py

```diff
@@ -77,8 +78,8 @@ def add_args(parser):
         parser.add_argument(
             "--silence-token", default="\u2581", help="token for silence (used by w2l)"
         )
-        parser.add_argument('--max-source-positions', default=2048, type=int, metavar='N',
-                            help='max number of tokens in the source sequence')
+        parser.add_argument('--max-source-positions', default=sys.maxsize, type=int, metavar='N',
+                            help='max number of frames in the source sequence')
         parser.add_argument('--max-target-positions', default=1024, type=int, metavar='N',
```

Comment:
We can make this default value as `sys.maxsize` as well, unless we know that there are no utterances in librispeech dataset with targets BPE units longer than 1024 .

Reply:
Thanks for your reviews! I guess it depends on the BPE size used. In my case, I am using 8000 BPE and the longest target contains 132 tokens. 1024 is the default max for the translation task too. I can change it to `sys.maxsize` if you prefer, but IMHO this is reasonable. Thanks.

Reply:
Thanks for checking!